ipatests/prci_definitions/gating.yaml

```diff
@@ -110,3 +110,15 @@ jobs:
         template: *ci-master-f27
         timeout: 3600
         topology: *master_1repl_1client
+
+  fedora-latest-ipa-4-11/test_ipalib_install:
```

Comment:
The definition needs to be replaced like this:
```
  fedora-27/test_ipalib_install:
    requires: [fedora-27/build]
    priority: 100
    job:
      class: RunPytest
      args:
        build_url: '{fedora-27/build_url}'
        test_suite: test_ipalib_install/test_kinit.py
        template: *ci-master-f27
        timeout: 3600
        topology: *master_1repl
```

I updated your PR with this change.